src/cascadia/TerminalApp/JsonUtils.h

```diff
@@ -574,6 +677,20 @@ namespace TerminalApp::JsonUtils
         GetValueForKey(json, key1, val1);
         GetValuesForKeys(json, std::forward<Args>(args)...);
     }
+
+    // SetValueForKey, type-deduced, manual converter
+    template<typename T, typename Converter>
+    void SetValueForKey(Json::Value& json, std::string_view key, const T& target, Converter&& conv)
+    {
+        json[key.data()] = conv.ToJson(target);

```

Comment:
is there a version of the json getter that can take a string view like the one in `GetValueForKey`?

Reply:
even if we have to manually use it